services/apps/alcs/src/alcs/application/application-decision-meeting/application-decision-meeting.service.ts

```diff
@@ -131,7 +131,17 @@ export class ApplicationDecisionMeetingService {
   > {
     return await this.appDecisionMeetingRepository
       .createQueryBuilder('meeting')
-      .select('reconsideration.uuid, MAX(meeting.date) as next_meeting')
+      .select('reconsideration.uuid', 'uuid')
+      .addSelect(
+        `
+        CASE
+          WHEN MIN(CASE WHEN meeting.date >= (CURRENT_DATE) THEN meeting.date END) is NOT NULL
```

Comment:
Have you made sure that `CURRENT_DATE` accounts for the time zone? I believe our dates are stored with time zones. This may work find, but I've run into issues with this before, so good to make sure.